tests/Invoke-DbaQuery.Tests.ps1

```diff
@@ -4,10 +4,10 @@ Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
 
 Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
     Context "Validate parameters" {
-        $paramCount = 13
+        $paramCount = 14
```

Comment:
Check #4695 and you might go ahead and adjust this test to match the new pattern for unit testing.

Reply:
I'll look into it.

Reply:
Done.